src/docs/gold_standard_slot_examples.md

```diff
@@ -0,0 +1,444 @@
+# Gold Standard MIxS LinkML Slots Examples
+
+## Introduction
+
+This page lists a range of 'gold standard' examples of slots that are used in the MIxS LinkML schema.
+
+There are examples of slots that cover a range of common types of data, including:
+
+- Basic slots (i.e., match exactly to a LinkML `range` type):
+  - [Integer](#integer) (numeric)
+  - [Float](#float) (numeric)
+  - [String](#string) (free text)
+  - [Fixed list](#fixed-list) (enumeration)
+  - [Boolean](#boolean)
+- Complex (i.e., common more complex `range` types that are more specific to MIxS):
+  - [Structured text (text only)](#structured-text-text-only) (text only)
+  - [Structured text (with unit, measurement)](#structured-text-numeric-with-unit-measurement) (numeric with units)
+  - [Structured text (ontology term)](#structured-text-ontology-term)
+  - [URL](#url) (or other link utilising a URL like structure)
+  - [Mixed type (free text or ontology)](#mixed-type) (either free text, or from an ontology)
+
+You can use these as templates on how to construct your own new slot to the highest quality level expected.
+
+## Overview
+
+Every MIxS LinkML slot should have at a minimum the following attributes:
+
+- [`description`](https://linkml.io/linkml/schemas/metadata.html#providing-descriptions): the description of what the metadata term is for
+- [`title`](https://linkml.io/linkml-model/latest/docs/title/): a short human readable 'title' for the slot
+- [`examples`](https://linkml.io/linkml-model/latest/docs/examples/): examples values demonstrating how the slot should be used
+- [`in_subset`](https://linkml.io/linkml-model/latest/docs/in_subset/): the section of the schema that the slot belongs based on a [fix list of MIxS categories](https://github.com/GenomicsStandardsConsortium/mixs/blob/609b0f567486f64cb7061246588d8006f87fa138/src/mixs/schema/mixs.yaml#L21-L26)
+  - Note: this system may be replaced in the near future!
+- [`keywords`](https://linkml.io/linkml-model/latest/docs/keywords/): useful keywords to allow grouping of related slots together
+- [`slot_uri`](https://linkml.io/linkml-model/latest/docs/slot_uri/): a unique ID assigned by MIxS
+  - This likely only gets assigned upon acceptance and merging by the core GSC MIxS team
+  - During development you can use a dummy value for this
+- [`range`](https://linkml.io/linkml/schemas/slots.html#ranges): specifying the type of 'object' that put in the slot (numeric, string, etc.)
+
+And for some slots, the following attributes are also recommended:
+
+- If a term should be either mandatory or optional
+  - [`recommended`](https://linkml.io/linkml/schemas/slots.html#recommended): a boolean value indicating if the slot is recommended be filled
+  - [`required`](https://linkml.io/linkml/schemas/slots.html#required): a boolean value indicating if the slot is mandatory to be filled
```

Comment:
I assume that we would never add any new terms with both recommended and required set. There are a few cases of that now that should be cleaned up. It is also possible that a term could be considered recommended in a Checklist and required in an Extension, so it would appear as both in a dynamically constructed combination class.

Reply:
I think that's what the "either mandatory or optional" part it.
That said, I think we should review the terms that are present in checklists and extensions... I assume that shouldn't happen and if it does that seems confusing

Reply:
Also, in terms of style, I don't think we should ever assert `false`. Any Boolean metaslot that isn't asserted is false by definition.

Reply:
Is the conclusion that what is added here is to just add 'recommended: true` if it is recommended etc., and just document you should never specify `recommended: false`?